### PR TITLE
doc_fragments: Clean up parameter types (cloud)

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
This PR includes:
- Parameter types added
- Copyright format fixes
- Short license statement
- Description fixes (only for a few files, then I stopped :-))

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
cloud module_utils

### CODE REVIEW

File: lib/ansible/plugins/doc_fragments/vultr.py [hunk 1]
```diff
@@ -1,44 +1,50 @@
...
     description:
       - API key of the Vultr API.
       - The ENV variable C(VULTR_API_KEY) is used as default, when defined.
+    type: str
   api_timeout:
     description:
       - HTTP timeout to Vultr API.
       - The ENV variable C(VULTR_API_TIMEOUT) is used as default, when defined.
       - Fallback value is 60 seconds if not specified.
+    type: str
```
Comment (resmo) [inline lib/ansible/plugins/doc_fragments/vultr.py]: this should be int IMHO

File: lib/ansible/plugins/doc_fragments/rackspace.py [hunk 2]
```diff
@@ -1,55 +1,49 @@
...
         I(username) are provided.
+    type: path
     aliases:
       - creds_file
   env:
     description:
       - Environment as configured in I(~/.pyrax.cfg),
         see U(https://github.com/rackspace/pyrax/blob/master/docs/getting_started.md#pyrax-configuration).
+    type: str
     version_added: 1.5
```
Comment (resmo) [inline lib/ansible/plugins/doc_fragments/rackspace.py]: quote?

File: lib/ansible/plugins/doc_fragments/rackspace.py [hunk 3]
```diff
@@ -1,55 +1,49 @@
...
     default: DFW
   username:
     description:
       - Rackspace username, overrides I(credentials).
+    type: str
   verify_ssl:
     description:
       - Whether or not to require SSL validation of API endpoints.
+    type: bool
     version_added: 1.5
```
Comment (resmo) [inline lib/ansible/plugins/doc_fragments/rackspace.py]: quote?

File: lib/ansible/plugins/doc_fragments/vultr.py (hunk 1 above)
Comment (dagwieers) [reply lib/ansible/plugins/doc_fragments/vultr.py]: Fixed.

File: lib/ansible/plugins/doc_fragments/rackspace.py (hunk 2 above)
Comment (dagwieers) [reply lib/ansible/plugins/doc_fragments/rackspace.py]: What do you mean ?

File: lib/ansible/plugins/doc_fragments/rackspace.py (hunk 3 above)
Comment (resmo) [reply lib/ansible/plugins/doc_fragments/rackspace.py]:
~~~diff
- version_added: 1.5
+ version_added: "1.5"
~~~
?

File: lib/ansible/plugins/doc_fragments/rackspace.py (hunk 2 above)
Comment (resmo) [reply lib/ansible/plugins/doc_fragments/rackspace.py]:
~~~diff
- version_added: 1.5
+ version_added: "1.5"
~~~
?

File: lib/ansible/plugins/doc_fragments/rackspace.py (hunk 3 above)
Comment (dagwieers) [reply lib/ansible/plugins/doc_fragments/rackspace.py]: Damn, to fix this for all files it's going to be hard to know which PR :-(

Comment (dagwieers) [reply lib/ansible/plugins/doc_fragments/rackspace.py]: Fixed.

File: lib/ansible/plugins/doc_fragments/rackspace.py (hunk 2 above)
Comment (dagwieers) [reply lib/ansible/plugins/doc_fragments/rackspace.py]: Fixed.

File: lib/ansible/plugins/doc_fragments/docker.py [hunk 4]
```diff
@@ -54,14 +54,14 @@ class ModuleDocFragment(object):
             - Path to the client's TLS certificate file.
             - If the value is not specified in the task and the environment variable C(DOCKER_CERT_PATH) is set,
               the file C(cert.pem) from the directory specified in the environment variable C(DOCKER_CERT_PATH) will be used.
-        type: str
+        type: path
```
Comment (felixfontein) [inline lib/ansible/plugins/doc_fragments/docker.py:57]: Why don't you also change `cacert_path` (one above)? Also, now the types differ from the ones here: https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/docker/common.py#L83-L85

Comment (dagwieers) [reply lib/ansible/plugins/doc_fragments/docker.py:57]: Fixed.

**Comment by dagwieers** [comment]
It helps if we wait for Shippable CI, since it verifies the types with the argspec.

**Review by resmo — APPROVED** [review]
shipit

**Comment by dagwieers** [comment]
cc @gundalow @acozine

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
rebuild_merge

**Pull request merged**